exp/query/api.go

```diff
@@ -145,6 +151,19 @@ func (a *API) Search(ctx context.Context, req *SearchRequest) (*SearchResponse,
 		return &resp, nil
 	}
 
```

Comment:
Good point. Yes. I already know it doesn't match feature parity with core Weaviate. Initially when I started it I thought it might be hard to support both in same query. But now I think about it, shouldn't be too hard.

I will either support in this same PR or add TODO to address in follow up PR.